.buildkite/runAllTests

```diff
@@ -3,7 +3,7 @@
 set -x
 
 # run in series due to saucelabs concurrency limits
-nightwatch --config nightwatch-ci.js --env chrome-mac
+nightwatch --config nightwatch.js --env chrome-mac
```

Comment:
Shouldn't we use the CI config for this?

Reply:
whoops - yeah, updated